opensearch/catalog-opensearch-endpoint/pom.xml

```diff
@@ -142,6 +147,8 @@
               org.objectweb.asm.tree,
               org.objectweb.asm.tree.analysis,
               org.objectweb.asm.util,
+              ddf.catalog.impl,
```

Comment:
The ddf.catalog.*.impl packages should be embedded and not imported.  Looks like the current imports are incorrect too.